pkgs/development/python-modules/asyncua/default.nix

```diff
@@ -20,7 +20,7 @@
 
 buildPythonPackage rec {
   pname = "asyncua";
-  version = "1.1.0";
+  version = "1.1.5";
   pyproject = true;
```

Comment:
```suggestion
  pyproject = true;
  disabled = pythonAtLeast "3.12";
```

Reply:
And pin opcua-client-gui to python 3.11 please.

Reply:
I'll disable the one failing test instead.